docs/index.md

```diff
@@ -30,90 +30,73 @@ The following 3 license types (`jq .type`) do **NOT** support APIs:
 - OSS
 
 ## Example Usage
-```hcl
+
+```terraform
 # Required for Terraform 0.13 and up (https://www.terraform.io/upgrade-guides/0-13.html)
 terraform {
   required_providers {
-    artifactory = {
-      source  = "registry.terraform.io/jfrog/projects"
-      version = "2.2.7"
+    project = {
+      source  = "registry.terraform.io/jfrog/project"
+      version = "0.0.2"
     }
   }
 }
 
-# Configure the Artifactory provider
-provider "artifactory" {
-  url = "${var.artifactory_url}/projects"
-  username = "${var.artifactory_username}"
-  password = "${var.artifactory_password}"
-}
-
-# Create a new repository
-resource "artifactory_local_repository" "pypi-libs" {
-  key             = "pypi-libs"
-  package_type    = "pypi"
-  repo_layout_ref = "simple-default"
-  description     = "A pypi repository for python packages"
-}
-```
+resource "project" "myproject" {
+  key = "myproj"
+  display_name = "My Project"
+  description  = "My Project"
+  admin_privileges {
+    manage_members   = true
+    manage_resources = true
+    index_resources  = true
+  }
+  max_storage_in_gibabytes   = 10
+  block_deployments_on_limit = false
+  email_notification         = true
 
-## Authentication
-The Artifactory provider supports multiple means of authentication. The following methods are supported:
+  member {
+    name  = "user1"
+    roles = ["developer","project admin"]
+  }
 
-    * Basic Auth
-    * Bearer Token
-    * JFrog API Key Header
+  member {
+    name  = "user2"
+    roles = ["developer"]
+  }
 
-### Basic Auth
-Basic auth may be used by adding a `username` and `password` field to the provider block
-Getting this value from the environment is supported with the `ARTIFACTORY_USERNAME` and `ARITFACTORY_PASSWORD` variable
+  group {
+    name = "dev-group"
+    roles = ["developer"]
```

Comment:
are you able to relate a member to a group? That'd be how to demonstrate

Reply:
Group's membership is not managed by this provider. Customers will have to managed the group's composition via either the UI/REST/TF RT provider separately.